### ISSUE TITLE
Prompted to log in even after editing cookies.txt

### ISSUE THREAD

**Issue by hobarrera** [Sude-/lgogdownloader#122]
I'm prompted to log in even after I posted my exported cookies. This is the file format I have:

```
# Cookies for domains related to gog.com.
# This content may be pasted into a cookies.txt file and used by wget
# Example:  wget -x --load-cookies cookies.txt https://www.gog.com/
#
login.gog.com	FALSE	/	TRUE	1519913329.513047	galaxy-login-tsa	REDACTED
.gog.com	TRUE	/	FALSE	0	cart_token	REDACTED
login.gog.com	FALSE	/	TRUE	1521641556.459701	galaxy-login-s	REDACTED
login.gog.com	FALSE	/	TRUE	1535897562.231164	galaxy-login-al	REDACTED
.gog.com	TRUE	/	FALSE	1535897565.479089	gog_lc	AR_USD_en-US
.gog.com	TRUE	/	FALSE	1535897565.479169	gog-al	REDACTED
.gog.com	TRUE	/	FALSE	0	gog_us	REDACTED
```

It's saved as `~/.config/lgogdownloader/cookies.txt`

I just get prompted over and over and login fails:

```
Email: hugo@barrera.io
Password: 
API: Login failed
```

I tried relogging a few times and updating the cookies, but it just won't work.

**Comment by Sude-** [comment]
I was able to replicate this issue.
It seems to be a server side issue that affect the API login.
Other API requests still return valid results when I have valid credentials.

Run ```lgogdownloader --login-api --verbose``` and check if the API response for login returns "500 Internal Server Error".
I suggest reporting this issue to GOG support.
I'll try couple changes to the login code and if the changes don't help then I'll report this to GOG support myself.

**Comment by hobarrera** [comment]
Yup, it's a 500. Though this is after being prompted to relogin.

**Comment by Sude-** [comment]
706158d makes the downloader use Galaxy API for getting game details and should allow most features to work without valid API login

**Comment by hobarrera** [comment]
I got an odd error when logging in, but then downloading worked for me. I'm _guessing_ this is fixed now?
